frontend/tests/pages/index.test.tsx

```diff
@@ -37,7 +37,7 @@ describe("Index", () => {
 
   it("passes accessibility scan", async () => {
     const { container } = render(<Index />);
-    const results = await axe(container);
+    const results = await waitFor(() => axe(container));
```

Comment:
Just curious why this was added for my own knowledge :D

Reply:
Because we're using the next/link Link component, that returns a promise, I think, so without the waitFor, the axe was not able to see the entire page. As a result it logged a bunch of warnings.